### PR TITLE
Add Coin98 Wallet Extensions

### PR DESCRIPTION
**Pull request by khiemsoft**
**Wallet description**
Coin98 Super Wallet is the leading multi-chain crypto & AI wallet, designed to seamlessly connect users to the crypto world safely and securely.

We now support 120+ networks, including EVMs and non-EVM chains. With 10M+ users in over 170 countries, Coin98 Super Wallet is fulfilling untapped demand and enhancing in-demand utilities in the DeFi space, helping people to access DeFi services effortlessly.

### CODE REVIEW

File: src/data/wallets/wallet-data.ts [hunk 1]
```diff
@@ -2083,6 +2084,63 @@ export const walletsData: WalletData[] = [
...
+    gas_fee_customization: true,
+    ens_support: true,
+    erc_20_support: true,
+    buy_crypto: true,
+    withdraw_crypto: false,
+    multisig: false,
+    social_recovery: false,
+    onboard_documentation: "https://docs.coin98.com/",
+    documentation: "https://docs.coin98.com/",
+    new_to_crypto: true,
```
[the remark this answers is not in the document]
Comment (konopkja) [reply src/data/wallets/wallet-data.ts:2142]: this metadata is only assigned by the core team upon careful ux review, not by wallet providers

Comment (minimalsm) [inline src/data/wallets/wallet-data.ts:2142]:
```suggestion
```

**Review by minimalsm — APPROVED** [review]

File: src/data/wallets/wallet-data.ts (hunk 1 above)
Comment (minimalsm) [inline src/data/wallets/wallet-data.ts:2142]: @khiemsoft please remove this field per @konopkja's comment. I don't have write access to this PR or I'd do it myself :)

Could you also clear merge conflicts and I'll get this merged?

Thanks 👍 

```suggestion
```

**Review by minimalsm — CHANGES_REQUESTED** [review]

File: src/data/wallets/wallet-data.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (khiemsoft) [reply src/data/wallets/wallet-data.ts]: @konopkja Already submitted your fixed suggestion, thanks for your review.

Comment (khiemsoft) [reply src/data/wallets/wallet-data.ts]: @minimalsm Already submitted your fixed suggestion; thanks for your review.

**Pull request closed without merging**